lib/kitchen/driver/ssh_base.rb

```diff
@@ -115,6 +115,14 @@ def verify(state) # rubocop:disable Metrics/AbcSize
       rescue Kitchen::Transport::TransportFailed => ex
         raise ActionFailed, ex.message
       ensure
+        if verifier[:scp_file]
```

Comment:
In fact I want to download junit.xml with detailed report of serverspec failures.